activities/ActivityTemplateVue.activity/js/activity.js

```diff
@@ -0,0 +1,107 @@
+// Rebase require directory
+requirejs.config({
+	baseUrl: "lib",
+	paths: {
+		activity: "../js"
+	}
+});
+
+// Vue main app
+let app = new Vue({
+	el: '#app',
+	components: {
+		'toolbar': Toolbar, 'toolbar-item': ToolbarItem, 'localization': Localization, 'tutorial': Tutorial,
+		'presence': Presence, 'journal': Journal
+	},
+	data: {
+		currentUser: {
+			user: {}
+		},
+		full: false,
+		context: null,
+		presence: null,
+		icon: null,
+		pawns: [],
+		l10n: {
+			stringHello: '',
+			stringPlayed: '',
+			stringAddPawn: ''
+		}
+	},
+	mounted: function () {
+		// this.presence = this.$refs.presence.presence;
+
+		var vm = this;
+		requirejs(["sugar-web/activity/activity", "sugar-web/env", "sugar-web/graphics/icon"], function (activity, env, icon) {
+			// Initialize Sugarizer
+			activity.setup();
+			
+			env.getEnvironment(function (err, environment) {
+				document.getElementById("user").innerHTML = "<h1>"+vm.l10n.stringHello+" "+environment.user.name+"!</h1>";
+				vm.currentenv = environment;
+			});
+
+			vm.icon = icon;
```

Comment:
Here again you could do a component instead

Reply:
Created a Pawn component which uses the icon library in a8302fe.